### PR TITLE
Milestone 2 Portfolio: Mobile version

### PR DESCRIPTION
**Pull request by mavericks-db**
## Mobile version
-replace icon bar with inline svg
-add svg hover effect
-add work and about me section

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,39 +1,230 @@
...
+    <div class="grid-container">
+      <div class="grid-item">
+        <img src="https://via.placeholder.com/300x250" alt="imagePlaceholder" />
+        <div class="description-container">
+          <h3>Multi-Post Stories Gain+Glory</h3>
+          <div class="tags">
+            <h4>Ruby on rails</h4>
+            <h4>CSS</h4>
+            <h4>JavaScript</h4>
+            <h4>HTML</h4>
```
Comment (thecodechaser) [inline index.html:96]: - [ ] You did a very good job with the style but for this project, you are supposed to stick to the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A988) as much as possible. Please use correct background-color for technologies `background-color: #EBF0EE;`

File: index.html [hunk 2]
```diff
@@ -1,39 +1,230 @@
...
+  </div>
+
+  <!-- About Me Section -->
+  <section class="aboutMeSection">
+    <h2 class="aboutMe">About Me</h2>
+    <p class="aboutMeText"> I can help you build a product, feature or website Look through some of my work and
+      experience! If you like what you see and have a project you need coded, don't hesitate to contact me.
+    </p>
+    <button type="button">Get My Resume</button>
+  </section>
```
Comment (thecodechaser) [inline index.html:181]: - [ ] Please fix the background images for the about section, They are overflowing on content.

### Your design

image

### Figma design

image

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @mavericks-db👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️

## Required Changes ♻️

- [ ] To make your README file more professional, Please add other sections like acknowledgment, getting started. You can use this [template](https://github.com/microverseinc/readme-template) but customize it for your project.

- [ ] Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by CollinsTatang — APPROVED** [review]
Hi @mavericks-db, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
image
### **OPTIONAL CHANGES**

- [x] Kindly fix your list of technologies to be as on the design.
**Your Design**
fixButton
**Figmal Design**
checks


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**